Makefile

```diff
@@ -6,7 +6,8 @@ test:
 	coverage run --branch --include=analytics/\* --omit=*/test* setup.py test
 
 release:
-	python setup.py sdist bdist_wheel upload
+	python3 setup.py sdist bdist_wheel
+	twine upload dist/*
```

Comment:
Is there a reason to use twine rather than the existing way?

Reply:
Yes. twine can take username/password via environment variables, as opposed specifying them (in plaintext) in .pypirc file as required by setup.py. And it used HTTPS as opposed to HTTP, but that's not a problem anymore in newer versions of Python.